src/fusion_power_plant.cc

```diff
@@ -12,45 +12,71 @@ std::string FusionPowerPlant::str() {
 
 // - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
 void FusionPowerPlant::EnterNotify() {
-  //Pseudocode implementation of EnterNotify()
   cyclus::Facility::EnterNotify();
-  fuel_usage_mass = (burn_rate * (fusion_power / MW_to_GW) / seconds_per_year * context()->dt());
-  fuel_usage_atoms = fuel_usage_mass / tritium_atomic_mass;
-  blanket_turnover = blanket_size * blanket_turnover_rate;
 
-  //Create the blanket material for use in the core
-  const cyclus::Composition::Ptr enriched_li = context->GetRecipe(blanket_inrecipe);
-  cyclus::Material::Ptr blanket = cyclus::Composition::CreateFromAtom(0.0, enriched_li);
+  //fuel_usage_mass = (burn_rate * (fusion_power / MW_to_GW) / 
+  //  seconds_per_year * context()->dt());
+  //fuel_usage_atoms = fuel_usage_mass / tritium_atomic_mass;
+  //blanket_turnover = blanket_size * blanket_turnover_rate;
```

Comment:
I'm assuming that we're just commenting these out until we actually get to a method that needs to use these variables.

Reply:
Yeah, I thought about deleting them, but figured since we'll likely add them all back in the next step it was fine to just leave them as comments.